doc/deprecated.md

```diff
@@ -50,6 +50,21 @@ namespace. For status on this see [#8234].
 
 </details>
 
+<details>
+<summary>Pubsub: remove legacy admin topic APIs</summary>
+<br>
+
+- On 2024-12-31 (or shortly after) will remove the hand-written versions of the
```

Comment:
None of the other warnings here bother with the (as you observed offline) mostly meaningless dates.  So, just _"We will remove the ..."_?

Reply:
Ack, done